src/serac/physics/materials/thermal_material_input.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2019-2023, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "serac/physics/materials/thermal_material_input.hpp"
+
+namespace serac {
+
+void ThermalMaterialInputOptions::defineInputFileSchema(axom::inlet::Container& container)
+{
+  // Define schema with each thermal material parameter
+  container.addString("model", "The model of material").required(true);
+  container.addDouble("density", "Initial mass density");
+  container.addDouble("cp", "The specific heat capacity");
+
+  // For LinearIsotropicConductor
+  container.addDouble("kappa", "The conductivity parameter");
+
+  // For LinearConductor
+  container.addInt("dim", "Dimension of conductivity tensor parameter");
+  auto& kappa_tensor_container = container.addStruct("kappa_tensor", "The conductivity tensor parameter");
+  kappa_tensor_container.addDoubleArray("row1", "First row of the conductivity tensor parameter");
+  kappa_tensor_container.addDoubleArray("row2", "Second row of the conductivity tensor parameter");
+  kappa_tensor_container.addDoubleArray("row3", "Third row of the conductivity tensor parameter");
```

Comment:
Leaving a comment here so that this is cleaned up after we have a better way of handling this in Inlet.